VocaDbWeb/package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "private": true,
+    "scripts": {
+        "dev": "npm run development",
+        "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+        "watch": "npm run development -- --watch",
+        "watch-poll": "npm run watch -- --watch-poll",
+        "hot": "cross-env NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
+        "prod": "npm run production",
+        "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+        "css-dev": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --env.mixfile=webpack.css.js --config=node_modules/laravel-mix/setup/webpack.config.js",
+        "css-prod": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --hide-modules --env.mixfile=webpack.css.js --config=node_modules/laravel-mix/setup/webpack.config.js"
+    },
+    "devDependencies": {
+        "cross-env": "^5.1",
+        "laravel-mix": "^4.0.7",
+        "laravel-mix-merge-manifest": "^0.1.2",
+        "less": "^3.10.3",
+        "less-loader": "^5.0.0",
+        "ts-loader": "^6.2.1",
+        "typescript": "^3.7.4",
+        "vue-template-compiler": "^2.6.11"
+    },
+    "dependencies": {
+        "bootstrap-sass": "^2.3.2",
+        "jquery": "^2.2.1",
+        "jquery-ui": "^1.10.4",
+        "knockout": "^3.4.1",
+        "knockout-punches": "^0.5.1",
+        "lodash": "^4.17.3",

```

Comment:
Since these JS libraries are now installed using npm (which is good), do you think we can remove them from nuget (packages.config)? JS packages used to be on nuget, but not anymore, because Microsoft is encouraging people to use npm for that.

Reply:
> Since these JS libraries are now installed using npm (which is good), do you think we can remove them from nuget (packages.config)?

Yes I think so, and I've uninstalled those NuGet packages.